src/sentry/integrations/gitlab/integration.py

```diff
@@ -128,12 +128,26 @@ class InstallationForm(forms.Form):
     group = forms.CharField(
         label=_("GitLab Group Path"),
         help_text=_(
-            "This can be found in the URL of your group's GitLab page. "
+            "This can be found in the URL of your group's GitLab page."
             "<br>"
-            "For example, if your group can be found at "
+            "For example, if your group URL is "
             "https://gitlab.com/my-group/my-subgroup, enter `my-group/my-subgroup`."
+            "<br>"
+            "Leave this empty to integrate an entire self-managed GitLab instance."
```

Comment:
We should require the group path when the instance is `gitlab.com` I don't think there is a use-case to work with all of gitlab.com.

Reply:
Special-casing `gitlab.com` to require a group path makes sense. I'll work on that.

Reply:
Added an error message that group path is required, if the url's host is gitlab.com or www.gitlab.com and group path is empty.

<img width="887" alt="Screen Shot 2019-10-21 at 4 11 49 PM" src="https://user-images.githubusercontent.com/27340/67247087-a15d8380-f41d-11e9-9afd-2eef50ca2d90.png">